CardIOSample/CardIOSample/DisplaySVViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DisplaySVViewController.swift
+//  CardIOSample
+//
+//  Created by Philip Sawyer on 5/26/16.
+//  Copyright © 2016 Philip Sawyer. All rights reserved.
+//
+
+import UIKit
+
+class DisplaySVViewController: UIViewController, CardIOViewDelegate {
+    
+    @IBOutlet weak var scanCardButton: UIButton!
+    @IBOutlet weak var resultsLabel: UILabel!
+    @IBOutlet weak var cancelButton: UIButton!
+    
+    var cardIOView : CardIOView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "CardIO as SubView"
+        
+        if !CardIOUtilities.canReadCardWithCamera() {
+            self.scanCardButton.hidden = true
+            self.resultsLabel.text = "Camera is not supported"
+        } else {
+            self.resultsLabel.hidden = true
+        }
+        self.cancelButton.hidden = true
+        
+        let rect = CGRect(x: 10, y: 80, width: 400, height: 400)
+        self.cardIOView = CardIOView(frame: rect)
+        cardIOView.delegate = self
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        CardIOUtilities.preload()
+    }
+    
+    func cardIOView(cardIOView: CardIOView!, didScanCard cardInfo: CardIOCreditCardInfo!) {
+        print("didScanCard")
+        
+        if cardInfo != nil {
+            let result = NSString(format: "Received card info.\n Number: %@\n expiry: %02lu/%lu\n cvv: %@.", (cardInfo.redactedCardNumber) ?? "", cardInfo.expiryMonth, cardInfo.expiryYear, (cardInfo.cvv) ?? "")
```

Comment:
You should be able to stay away from making explicit `NSString`s. You can instead use Swift strings and use its form of string interpolation:
```swift
"Here is a string \(someVariable)"
```